ContextDesktop/src/main/java/com/github/migi_1/Context/ServerState.java

```diff
@@ -0,0 +1,57 @@
+package com.github.migi_1.Context;
```

Comment:
Good point.
Wait.. who are you? What are you doing here? GET BACK TO YOUR OWN PROJECT! PLAGIARIASM! PLAGIARIASM! PLAGIARIASM!